web/src/components/CustomThumbnail.tsx

```diff
@@ -0,0 +1,50 @@
+import { styled } from '@mui/material/styles'
+import React, { ReactElement } from 'react'
+
+const ThumbnailWrapper = styled('div')`
+  position: relative;
+  height: clamp(120px, 14vh, 160px);
+  width: 100%;
+  flex-shrink: 0;
+  border-radius: 8px;
+  overflow: hidden;
+
+  ${props => props.theme.breakpoints.down('md')} {
+    order: 1;
+    margin-top: 12px;
+  }
+`
+
+const Thumbnail = styled('img')<{ isPng?: boolean }>`
```

Comment:
🔧 The naming is not correct anymore
```suggestion
const Thumbnail = styled('img')<{ opaque?: boolean }>`
```